### ISSUE TITLE
Tab indicators remain visible over URL bar

### ISSUE THREAD

**Issue by Strigifowl** [zen-browser/desktop#3859]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Ever since the new update I've been using the single toolbar mode and enjoyed it. However for whatever reason the tab indicators remain visible over the URL bar instead of getting covered properly as can be seen in the video below.

https://github.com/user-attachments/assets/1a307af6-d2c9-473e-a264-7ad529725462

I am using some mods in this clip but it remains even on a default version. Any help would be appreciated.

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.2 

### What platform are you seeing the problem on?

Linux

**Comment by syscat29** [comment]
Hi, are you using any custom CSS?

**Comment by Strigifowl** [comment]
Not to my knowledge no

**Comment by mr-cheffy** [comment]
Any mods installed? That does not look like default zen

**Comment by Strigifowl** [comment]
Yes but it happens even when mods are turned off as can be seen here

https://github.com/user-attachments/assets/72b5febd-c2a9-4890-925c-1db0f4dea27d

**Comment by syscat29** [comment]
hmm that's weird, we tested in linux both stable and twilight and this does not happen

**Comment by pawelkuzia** [comment]
Same issue for me, with or without custom CSS an Zen Modes, both on stable and twilight.

EDIT
I've created a new profile and the problem is not present.

**Comment by Strigifowl** [comment]
I have tried the same and have freshly installed the browser previously and the issue remains for me.

**Comment by syscat29** [comment]
> I have tried the same and have freshly installed the browser previously and the issue remains for me.

Can you please try creating a new profile and check if the issue persists?

**Comment by syscat29** [comment]
**Update:** Have been trying to replicate this with no success so far, stable and twilight both behave as they should.

Please just like i said before, make sure no mods, custom css or extensions that could modify the t heme are loaded, try creating a new profile and see if the issue persists, please report back once you've tested these.

**Comment by Strigifowl** [comment]
> **Update:** Have been trying to replicate this with no success so far, stable and twilight both behave as they should.
> 
> Please just like i said before, make sure no mods, custom css or extensions that could modify the t heme are loaded, try creating a new profile and see if the issue persists, please report back once you've tested these.

How would I make sure I have no custom css loaded? Beyond that I can confirm that the issue persists even with no extensions, themes, or mods loaded on a fresh profile. This video is off a profile I created just this moment.

https://github.com/user-attachments/assets/6d50c7ee-970b-4fa7-b73c-9e35b0d9d613

**Comment by mr-cheffy** [comment]
Woah, this is truly a one time thing

**Comment by Strigifowl** [comment]
The issue is still relevant as of version 1.7.2b

**Comment by Strigifowl** [comment]
The issue has been solved with version 1.8.1b

**Comment by moderation** [comment]
Looks like 1.8.1b has been removed - https://github.com/zen-browser/desktop/releases. Lost all of my workspace tabs
